events/country-map.js

```diff
@@ -75,5 +75,6 @@ module.exports = {
   VE: 'Latin America',
   AU: 'South Pacific',
   NZ: 'South Pacific',
-  SG: 'South Pacific'
+  SG: 'South Pacific',
+  MK: 'Republic of Macedonia'
```

Comment:
Just add this pls. It should be Europe not Republic of Macedonia I guess. Also plese add it in alphabetical order in the europe group.